tests/plugins/test_add_labels.py

```diff
@@ -202,7 +202,6 @@ def test_add_labels_plugin(tmpdir, docker_tasker,
     ('vcs-url', DOCKERFILE_GIT),
     ('vcs-ref', DOCKERFILE_SHA1),
     ('com.redhat.build-host', 'the-build-host'),
-    ('Build_Host', 'the-build-host'),
```

Comment:
auto_labels should use only the new Labels, and old alias names are sorted in add_aliases